src/Digipolis.Delegation/Middleware/DelegationMiddleware.cs

```diff
@@ -65,6 +65,8 @@ public DelegationMiddleware(RequestDelegate next, ILogger<DelegationMiddleware>
                         ClaimsIdentity claimsIdentity = principal.Identities?.FirstOrDefault();
 
                         if (claimsIdentity != null && delegationUser.TrySetValues(claimsIdentity, token)) delegationUserParsed = true;
+                        
+                        //context.Request.Headers.Add(options.Value.DelegationHeader, "Bearer " + token);
```

Comment:
Misschien die regel gewoon verwijderen indien niet nodig. Dan vermijdt dit mogelijke verwarring in de toekomst.